lib/dl_api_lib/dl_api_lib/app/control_api/resources/connections.py

```diff
@@ -128,9 +128,11 @@ def post(self):  # type: ignore  # TODO: fix
         us_manager = self.get_us_manager()
 
         conn_availability = self.get_service_registry().get_connector_availability()
-        conn_type_is_available = conn_availability.check_connector_is_available(
-            ConnectionType[request.json.get("type")]  # type: ignore  # 2024-01-24 # TODO: Item "None" of "Any | None" has no attribute "get"  [union-attr]
-        )
+        conn_type = request.json and request.json.get("type")
+        if not (conn_type and conn_type in ConnectionType):
```

Comment:
better to expand brackets (`not conn_type or conn_type not in ConnectionType`) for readability

Reply:
fixed